### PR TITLE
chore: update owners

### PR DESCRIPTION
**Pull request by mikebz**

### CODE REVIEW

File: OWNERS_ALIASES
Comment (nan-yu) [inline OWNERS_ALIASES:1]: We also need to add [Camila-B](https://github.com/Camila-B) to the maintainers list?

Comment (mikebz) [reply OWNERS_ALIASES:1]: sure, but let's not stop this PR and just do a separate one for her when she is ready.

**Review by sdowell — APPROVED** [review]
/lgtm

**Pull request merged**